src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -276,8 +292,8 @@ const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holyTosca,
-    marginRight: 11,
+    color: COLORS.holytosca,
+    marginRight: dimen.normalizeDimen(11),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been updated and the margin right value is now dynamic. Ensure that `COLORS.holytosca` is correctly defined in your colors utility file and `dimen.normalizeDimen(11)` returns the expected result.
```diff
- color: COLORS.holyTosca,
- marginRight: 11,
+ color: COLORS.holytosca,
+ marginRight: dimen.normalizeDimen(11),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->